src/components/CodeBox.vue

```diff
@@ -121,7 +148,7 @@ watch(
         <b-form-input id="error-fix" v-model="currentEditingBug.correctValue"></b-form-input>
       </b-form-group>
 
-      <b-form-group label="Select ErrorType" label-for="error-type">
+      <b-form-group label="Select ErrorType" label-for="error-type" v-if="!isWordSpace(currentEditingBug.wordId)">
```

Comment:
I am not quite sure: Why is that not shown for between-words bugs?

Reply:
What error type is for example a missing `final`? I can understand that you want to specify a missing `;` as syntax, but I don't know if this is generally possible... What do you think @madebyTimo ?

Reply:
`Style`.
We could also implement it this way: The error type popup will only not be shown when the lecturer created the bug with `ErrorType.UNDEFINED`.
This should be applicable for all cases.

Reply:
> `Style`. We could also implement it this way: The error type popup will only not be shown when the lecturer created the bug with `ErrorType.UNDEFINED`. This should be applicable for all cases.

We can not do this. The user should not know the solution in the frontend before he submitted in the future when the backend is implemented.

Reply:
Why? I mean that would be possible, but it sounds rather complicated.
I'd recommend simply sending the solution for the next code sample when querying the next sample.
Will make things way easier and smoother.

Reply:
I think in future it will be possible for the lecturer to define if the error-type should be asked or not but I agree to @maexled we won't do it this way. If it is better to ask for error-type on a missing-bug or not is a hard question for me. If I got it right the error type only can be `Style` or `Syntax`. I think the question for the error-type can be omitted but if you see an additional value in asking for it we could add it.

Reply:
@delvh We planned to send the next code samle with the solution of the actual code sample. I think this is a way better solution and cause we considered these solution from the beginning it won't add extra-work